lab-yunjoo/app/lib/random-array-item.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+module.exports = function(array){
+  if(array.length === 0) return null;
+  let index = Math.floor(Math.random()*array);
```

Comment:
This doesn't work--it's being multiplied by the array, which is meaningless (and I would have thought would throw an error). This function also doesn't return anything.